static/js/lib/validation.js

```diff
@@ -48,11 +48,6 @@ export const validEmail = R.compose(
   R.trim
 )
 
-export const validNotMIT = R.compose(
```

Comment:
So it looks like we're losing this now, which I think was meant to require MIT users to always login via touchstone, but now if they don't already have an account, they don't have to. It doesn't look like registration was guarded by this, so this "require MIT users to login via touchstone" thing is basically entirely going away here.

Reply:
Ah, good catch. I forgot about the register page. I'll add this back in and add a separate validation function for the register page since it will be different from the login page now